pkg/operator/controllers/pullsecret/pullsecret_controller.go

```diff
@@ -184,15 +184,18 @@ func (r *Reconciler) ensureGlobalPullSecret(ctx context.Context, operatorSecret,
 		// allows for simpler logic flow, when delete and create are not handled separately
 		// this call happens only when there is a need to change, it has no significant impact on performance
 		err := r.client.Delete(ctx, secret)
+		r.log.Info("Pullsecret Not Found, Creating Again")
 		if err != nil && !kerrors.IsNotFound(err) {
 			return nil, err
 		}
 
 		err = r.client.Create(ctx, secret)
+		r.log.Info("Pullsecret Created")
```

Comment:
Same comment as above.

Reply:
It can become misleading if there was an error while creating the secret and still `Pullsecret Created` message is provided. Maybe check for `err != nil` and return error in that case and print the success message only if there's no error.

Reply:
You're right. I've added the suggested changes.